nautobot/docs/models/extras/relationship.md

```diff
@@ -9,11 +9,40 @@ Sometimes it is desirable to create a new kind of relationship between one (or m
 
 ## Relationship Filters
 
-Filters can be defined to restrict the type or selection of objects for either side of the connection. From the FEX example above, you can restrict the FEX side of the connection to only of devices with the FEX role, and restrict the controller side to only devices with Controller roles.
+Filters can be defined to restrict the type or selection of objects for either side of the connection. 
+
+### Relationship Filter Example
+
+As an example, let's create a relationship between Circuits and Devices. 
+In our situation we only would terminate Circuits on Devices with the Role of `edge`.
+
+To prevent the Circuit Relationship from showing up on any other Device, use a JSON filter to 
+limit the Relationship to only Devices with Role of `edge`:
+
+```
+{
+    "role": [
+        "edge"
+    ]
+}
+```
+
+For context, here is an image of the entire Relationship:
+
+![Image of Relationship with json filter](../../img/relationship_w_json_filter.png)
+
+Now, the Circuit Relationship field will show up on a Device with an `edge` role:
+
+![Image of Edge Device Relationships](../../img/edge_dev_ckt_relationship.png)
```

Comment:
`ckt` is not immediately obvious to me. Why not `edge_device_circuit_relationship.png` and `leaf_device_no_circuit_relationship.png` as filenames?